DeDriftAndResample/DeDriftAndResamplePipeline.sh

```diff
@@ -27,1094 +27,616 @@
 # <!-- References -->
 # [HCP]: http://www.humanconnectome.org
 #
+pipedirguessed=0
```

Comment:
`set -eu` appears to be missing (it is usually just after `#!/bin/bash`), please add it and test again.